bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BridgeHandler.java

```diff
@@ -190,8 +193,10 @@ private void scheduleInitialAccess(BoschHttpClient httpClient) {
      * to check if access if possible
      * pairs this Bosch SHC Bridge with the SHC if necessary
      * and starts the first log poll.
+     * <p>
+     * This method is package-protected to enable unit testing.
      */
-    private void initialAccess(BoschHttpClient httpClient) {
+    /* package */ void initialAccess(BoschHttpClient httpClient) {
```

Comment:
Left-over?

Reply:
I usually add this comment to indicate that the member is package protected on purpose (and not by accident). I could not find a coding style guide for this aspect in the OpenHAB developer documentation. I can also simply remove `/* package */` if this is preferred.

Reply:
If there is no code style rule against this and you prefer it, let's leave it.